### PR TITLE
docs: explain how to start PostgreSQL

### LINKED ISSUE #4964: Document that Postgres isn't started automatically
### Description

The readme file of Ubuntu images documents that MySQL must be started manually, but it's also the case for Postgres and it's not documented.
Fixed by #4909.

### Virtual environments affected

- [X] Ubuntu 18.04
- [X] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11
- [ ] Windows Server 2016
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

All.

### Is it regression?

_No response_

### Expected behavior

_No response_

### Actual behavior

_No response_

### Repro steps

n/a

### PR DESCRIPTION
**Pull request by dunglas**
# Description
Explain how to start PostgreSQL, as done for MySQL.

Closes #4964.

### CODE REVIEW

**Comment by nikolai-frolov** [comment]
Hi @dunglas, we don't update *-Readme.me files manually. All of them are generated automatically. Also please attach an issue for which these changes are.

**Comment by dunglas** [comment]
@nikolai-frolov done

File: images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1 [hunk 1]
```diff
@@ -33,6 +33,21 @@ function Get-SqlPackageVersion {
     return "SqlPackage $sqlPackageVersion"
 }
 
+function Build-PostgreSqlSection {
+    $output = ""
+
+    $output += New-MDHeader "PostgreSQL" -Level 4
+    $output += New-MDList -Style Unordered -Lines @(
+        (Get-PostgreSqlVersion ),
```
Comment (miketimofeev) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1:41]: nitpick
```suggestion
        (Get-PostgreSqlVersion),
```

**Comment by miketimofeev** [comment]
@dunglas could you please pull the latest main? Ubuntu-1804 fails for some reason

**Review by al-cheb — DISMISSED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Comment by miketimofeev** [comment]
@dunglas thank you for your contribution! The readme will be updated next week.

**Pull request merged**